src/Button/index.jsx

```diff
@@ -127,15 +129,17 @@ Button.defaultProps = {
   children: undefined,
   cssMap: undefined,
   iconPosition: "left",
-  iconType: "none",
+  iconType: undefined,
```

Comment:
I think this should probably be `"none"` as before:
```suggestion
  iconType: "none",
```